app/code/Shiphawk/Shipping/Model/Carrier.php

```diff
@@ -223,58 +223,45 @@ protected function _get($jsonRateRequest) {
         return $arr_res;
     }
 
-    public function getItems($request)
-    {
+    public function getItems($request) {
         $items = array();
 
+        // getAllItems() return all request items including clild items for :bundle products.
+        // So we have those check to include only parent items in rate request.
         foreach ($request->getAllItems() as $item) {
-
             if ($item->getProductType() != 'simple') {
-
-                $product = $this->objectManager->create('Magento\Catalog\Model\Product')->load($item->getProductId());
```

Comment:
There is not difference in getting the product with new or old version, but the main change here is related to getting the price from the `$item` directly, instead of getting the price from the product. Experimental way I determined that `$item->getPrice()` approach works for all (simple, configurable and bundle) item types.